### PR TITLE
Microsoft usgov support

### PR DESCRIPTION
**Pull request by kobymeir**
## Status
In Progress

## Related Issues
relates: https://jira-hq.paloaltonetworks.local/browse/CIAC-818

## Description
Adding a table indication support of National Clouds in Microsoft integrations.

### CODE REVIEW

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 1]
```diff
@@ -189,57 +190,55 @@ After you a redirected to the next page, in the **Overview** tab you will find y
 
        ![app-connect](../../../docs/doc_imgs/tutorials/tut-microsoft-auth-guide/app-connect.png)
 
-
-   2. Copy the ***AUTH_CODE*** (without the "code=" prefix). This value must be used in the MS Graph User in Cortex XSOAR integration in the **Authorization Code** field. 
-   
+   2. Copy the ***AUTH_CODE*** (without the "code=" prefix). This value must be used in the MS Graph User in Cortex XSOAR integration in the **Authorization Code** field.
 5. Add the information to the instance in Cortex XSOAR by going to Settings>Integrations>Microsoft Graph User>Add Instance.
 
    1. In the ***ID*** parameter field, type the client ID.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:196]:
```suggestion
   1. In the **ID** parameter field, type the client ID.
```

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 2]
```diff
@@ -189,57 +190,55 @@ After you a redirected to the next page, in the **Overview** tab you will find y
...
    2. in the ***Token*** parameter field, type the tenant ID.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:197]:
```suggestion
   2. in the **Token** parameter field, type the tenant ID.
```

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 3]
```diff
@@ -189,57 +190,55 @@ After you a redirected to the next page, in the **Overview** tab you will find y
...
    3. In the ***Key*** parameter field, type your client secret.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:198]:
```suggestion
   3. In the **Key** parameter field, type your client secret.
```

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 4]
```diff
@@ -189,57 +190,55 @@ After you a redirected to the next page, in the **Overview** tab you will find y
...
    4. Click the **Use a self-deployed Azure application** checkbox.
-   5. In the ***Redirect URI*** field, type the redirect URI we entered in the Azure portal.
+   5. In the ***Redirect URI*** field, type the redirect URI we are entering at the Azure portal.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:200]:
```suggestion
   5. In the **Redirect URI** field, type the redirect URI we are entering at the Azure portal.
```

**Comment by ShirleyDenkberg** [comment]
@michal-dagan Doc review completed.

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 5]
```diff
@@ -77,21 +77,24 @@ To configure a Microsoft integration that uses this authorization flow with a se
...
+  - For Microsoft Defender select the appropriate cloud using the *Endpoint Type* parameter.
+  - For using self-deployment option please select the *Custom* option and follow the instructions below.
+
+- Some of the Cortex XSOAR-Microsoft integrations support the deployment of national clouds through the self-deployed
  authorization flow. For more information about Microsoft National Clouds, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/graph/deployments).
- Currently, the following integration supports national clouds:
- * O365 Outlook Mail (Using Graph API)  
- 
-In order to use a national cloud, change the *URL* parameter to the corresponding address of the national cloud you are using. For example, see [O365 Outlook Mail (Using Graph API) - National Clouds](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-mail#using-national-cloud), for the supported cloud endpoints in this integration.
+ In order to use a national cloud, change the *URL* parameter to the corresponding address of the national cloud you are using.
```
Comment (michal-dagan) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:89]: maybe we need to change the display name of `azure_ad_endpoint` in "Azure Kubernetes Services" to `Server URL`
```suggestion
 In order to use a national cloud, change the *Server URL* parameter to the corresponding address of the national cloud you are using.
```

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 6]
```diff
@@ -77,21 +77,24 @@ To configure a Microsoft integration that uses this authorization flow with a se
 7. Save the instance.
 8. Run the `!<integration command prefix>-generate-login-url` command in the War Room and follow the instructions. For example, for Microsoft Graph User: `!msgraph-user-generate-login-url`.
 
-#### Using National Cloud
-Some of the Cortex XSOAR-Microsoft integrations support the deployment of national clouds through the self-deployed
+### Using National Cloud
```
Comment (michal-dagan) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:80]: I think you can move the paragraph to be first under "Self Deployed Application"

**Review by michal-dagan — APPROVED** [review]

**Pull request merged**